iceberg/iceberg-handler/src/test/queries/positive/fastforward_iceberg_branch.q

```diff
@@ -0,0 +1,23 @@
+-- SORT_QUERY_RESULTS
+set hive.explain.user=false;
+set hive.fetch.task.conversion=more;
+
+create external table ice01(a int, b string, c int) stored by iceberg stored as orc tblproperties ('format-version'='2');
+
+insert into ice01 values (1, 'one', 50), (2, 'two', 51), (111, 'one', 55);
+
+select * from ice01;
+
+-- create a branch named test1
+alter table ice01 create branch test1;
+
+select * from default.ice01.branch_test1;
+
+-- insert into main branch
+insert into ice01 values (10, 'ten', 53), (11, 'eleven', 52), (12, 'twelve', 56);
+select * from ice01;
+
+-- fastforward the branch
+explain alter table ice01 execute fastforward 'test1' to 'main';
+alter table ice01 execute fastforward 'test1' to 'main';
```

Comment:
Can we also add a negative qtest, which verifies that fastforward will fail if target is not an ancestor of source.
https://github.com/apache/iceberg/blob/181d3e21efaf287abdbaf2897b456fe44805d24f/api/src/main/java/org/apache/iceberg/ManageSnapshots.java#L175-L178

Reply:
will add one